src/features/ticketing/quantityField.tsx

```diff
@@ -0,0 +1,28 @@
+import { appSelector, useAppDispatch } from '../../app/hooks'
+import { FormControl, Input } from '@material-ui/core'
+import { setQty, selectTicketQty } from './ticketingSlice'
+import eventPageStyles from '../events/eventPageStyles'
+
+const QuantityField = () => {
+    const classes = eventPageStyles()
+    const dispatch = useAppDispatch()
+    const amount = appSelector(selectTicketQty)
+
```

Comment:
An entire file for a single element feels like too much

Reply:
This is now in Event Page again